Bearded.Graphics/Bearded.Graphics.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <LangVersion>8</LangVersion>
+    <LangVersion>latest</LangVersion>
     <Nullable>annotations</Nullable>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Nullable>enable</Nullable>
-    <TargetFrameworks>net5.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Isn't removing .NET 3.1 support a breaking change?

If we remove the .NET 3.1 support from here, we should also stop the GitHub Actions from using the 3.1 .NET version in its build and publish matrix.

#59 already replaced .NET 5 with .NET 6 so you can just look in that PR to see where to change things in the workflow files. (You'll probably need to merge/rebase this PR as well to get those changes.)

Reply:
Documented .net 3.1 removal above. I don't see a reason to still support it, and we use new features.

I merged master - do I need to do anything else?

Reply:
As discussed offline: 2961bedce7e76eaa664b5079fab72c0d640d28ce